doc/content/integrations/cloud-integrations/thinger-io/_index.md

```diff
@@ -0,0 +1,171 @@
+- **JSON Object**: If the plugin downlink request contains a JSON payload in the POST call, this function will receive this payload instead of the one configured in the device `downlink` property.
+
+The output of this method should be one of the following:
+
+- **Base64 String**: With binary information that can be sent directly to {{% tts %}} network. It is required if your {{% tts %}} application is not defining a converter.
+- **JSON Object**: If {{% tts %}} application provides a converter for your payloads, this method can return a JSON object that will be accesible in the converter method. In this case, creating a custom downlink method will be redundant, so create the funtion in {{% tts %}}, or in the plugin.
+
+Example of a downlink method converting a JSON device configuration into base64 as required by {{% tts %}}:
+
+```javascript
+/* convert a JSON object with the device configuration in a base64 string expected by {{% tts %}} */
+module.exports.downlink = function(payload){
+    let bytes = [];
+    bytes[0] = payload.enabled ? 1 : 0;
+    bytes[1] = payload.frequency;
+    bytes[2] = payload.threshold;
+    return Buffer.from(bytes).toString('base64');
+};
+```
+
+{{< note >}}
+The downlink method should return a base64 string if {{% tts %}} application does not define a converter.
+{{</ note >}}
+
+{{< /tabs/tab >}}
+
+{{< /tabs/container >}}
+
+## {{% tts %}} Console Configuration
+
+### Uplink Configuration
+
+The last tab of the plugin configuration interface is called `Webhook Settings`, it has been created to help the developers to complete the integration in {{% tts %}} Console, by providing all the information required to setup the webhook profile. 
+
+{{< figure src="thingerio_webhook_settings.png" alt="[Thinger.io](https://thinger.io/) {{% tts %}} Plugin webhook settings tab" >}}
+
+{{< note >}}
+The REST API does not define the application ID, this parameter will be checked by the plugin software to manage the payload according to the configuration.
+{{</ note >}}
+
+To create a new webhook integration follow the next steps in {{% tts %}} web console:
+
+1. Select the Application to be integrated.
+2. In the main menu open the `Integrations` section and click the `Webhooks` option. The webhooks list will be shown.
+
+{{< figure src="thingerio_tts_sidebar.png" alt="{{% tts %}} sidebar" >}}
+
+3. Clicking the **+Add webhook** blue button in the right top corner of the interface allows choosing between different webhooks integration templates. Select [Thinger.io](https://thinger.io/) template. Then, configure the webhook only requires filling the form with the information provided by [Thinger.io](https://thinger.io/) `Webhook Settings` tab and selecting JSON webhook format.
+
+{{< figure src="thingerio_tts_webhook.png" alt="[Thinger.io](https://thinger.io/) {{% tts %}} webhook settings" >}}
+
+{{< note >}}
+The Authorization header must be set up using the access token including "Bearer" command
```

Comment:
```suggestion
The **Authorization Header** must be set up using the access token including `Bearer` command.
```

Reply:
You can best spot the difference of using **this** and `this` on this example. For buttons, tabs, and field names in the Console we use **this**, while for variable values just like this one we use `this`. This is why I added some of these suggestions in the rest of the doc.